docs/src/data/hid.js

```diff
@@ -5896,7 +5896,7 @@ export default [
     ],
     documentation: "https://usb.org/sites/default/files/hut1_2.pdf#page=137",
     os: {
-      windows: null,
+      windows: false,
```

Comment:
```suggestion
      windows: true,
```
Thanks for responding @KingCoinless, maybe we can try just accepting this suggestion to fix this PR?

Reply:
Sure, I'll commit your suggestion. I haven't worked with Github in a while, so let me know if anything is amiss.